crates/ruff/src/registry.rs

```diff
@@ -318,40 +319,41 @@ ruff_macros::register_rules!(
     rules::pyupgrade::rules::ReplaceStdoutStderr,
     rules::pyupgrade::rules::RewriteCElementTree,
     rules::pyupgrade::rules::OSErrorAlias,
-    rules::pyupgrade::rules::RewriteUnicodeLiteral,
-    rules::pyupgrade::rules::RewriteMockImport,
+    rules::pyupgrade::rules::UnicodeKindPrefix,
+    rules::pyupgrade::rules::DeprecatedMockImport,
     rules::pyupgrade::rules::RewriteListComprehension,
-    rules::pyupgrade::rules::RewriteYieldFrom,
+    rules::pyupgrade::rules::YieldInLoop,
```

Comment:
I think either `YieldInForLoop` or probably rather `IterateAndYield`. Loop is too generic (since it doesn't apply to while loops).